modules/multi-packit.nix

```diff
@@ -18,14 +19,15 @@ let
   '';
 
   # Ports get assigned sequentially, starting at 8000 for outpack, 8080 for
-  # packit-api, and 8160 for packit-api's management interface.
+  # packit-api, 8160 for packit-api's management interface and a single 
+  # orderly runner api at port 8240 for all the instances.
   ports = lib.listToAttrs (lib.imap0
     (idx: name: lib.nameValuePair name {
       outpack = 8000 + idx;
       packit-api = 8080 + idx;
       packit-api-management = 8160 + idx;
     })
-    cfg.instances);
+    cfg.instances) // { orderly-runner-api = config.services.orderly-runner.port; }; # 8240 by default
```

Comment:
This `ports` object is only needed because we have an arbitrary number of port numbers.
Its "schema" at the moment is `ports :: InstanceName -> ServiceName -> PortNumber`.

Adding `orderly-runner-api` in there is just confusing IMO. You can just use `config.services.orderly-runner.port` whereever it is needed.